editors/liveview/assets/css/app.css

```
@@ -3,3 +3,551 @@
+  vertical-align: middle;
+}
+.dock-pane { flex: 1; min-height: 0; display: flex; flex-direction: column; overflow: auto; }
+.dock-pane[hidden] { display: none; }
+.ws-pane { padding: 10px 12px; }
+.ws-pane > form { flex: 1; min-height: 0; }
+.ws-wrap { flex: 1; min-height: 0; border: 1px solid var(--line); border-radius: 7px; }
+.ws-result {
+  margin-top: 8px; padding: 8px 12px; border: 1px solid var(--line); border-radius: 7px;
+  background: var(--code-bg); font-family: var(--code-font, ui-monospace, monospace);
+  font-size: 12.5px; display: flex; align-items: flex-start; gap: 8px; flex: none;
+  white-space: pre-wrap; word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)